### DISCUSSION TITLE
`<details>` doesn't render a collapsed section, but should.

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [damionx7/Zettel-Notes-Documentation#328] [Feature Requests]

**Comment by damionx7** [comment]
#356

**Comment by nathanl** [comment]
For those not familiar with it, `<details>` is this: https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/details

![details element](https://github.com/user-attachments/assets/41144051-52a6-4d5d-9bd7-4174dfc69701)

**Comment by damionx7** [comment]
I can implement a custom implementation, where clicking on `details` will show a text box with summary.

**Reply by nathanl** [reply]
I'm not sure how relevant this is, but `<details>` doesn't render properly in HTML emails, at least in Gmail (it shows only the contents of `<summary>` with no indication that it was ever expandable). So anyone who, like me, may want to [copy and paste a note's HTML into an email to share it](https://github.com/damionx7/Zettel-Notes-Documentation/discussions/358) would not see these render properly.

**Reply by roke-julian-lockhart** [reply]
> [`#discussioncomment-12922975`](https://github.com/damionx7/Zettel-Notes-Documentation/discussions/328#discussioncomment-12922975)

@nathanl, that seems like an (ultimately temporary) deficiency in *some, current* e-mail clients. [^1] [^2] Whether it applies to more standard-conformant ones, like Thunderbird and Proton Mail, isn't even yet known to us.

Consequently, if that's a concern of yours, the solution is surely to not utilise the element, rather than reconsider its implementation...?

[^1]: [`github.com/discourse/discourse/pull/7597#issuecomment-495688138`](https://github.com/discourse/discourse/pull/7597#issuecomment-495688138)

[^2]: [`learn.microsoft.com/en-au/answers/questions/2113739`](https://learn.microsoft.com/en-au/answers/questions/2113739/html-tag-(details-)-and-(summary-)-not-working-in?source=docs)

**Reply by nathanl** [reply]
Yep, you're right @RokeJulianLockhart

**Comment by damionx7** [comment]
I have added this feature,  where clicking on details will show a text box with summary.
